src/app/components/Checkbox/index.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { ComponentProps } from 'react';
+import { Story } from '@storybook/react';
+
+import { Checkbox } from './index';
+
+export default {
+  title: 'Atoms/Checkbox',
+  component: Checkbox,
+};
+
+const Template: Story<ComponentProps<typeof Checkbox>> = args => (
+  <Checkbox {...args} />
+);
```

Comment:
Having a second interactive template as well might be good. So a template that handles the checked state.